lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -469,6 +469,38 @@ CRI_HMRC_KBV_J6:
     next:
       targetState: PYI_NO_MATCH
 
+# No photo id M2B journey (J7)
+CRI_BANK_ACCOUNT_J7:
+  response:
+    type: cri
+    criId: bav
+    context: bank_account
```

Comment:
Whats `context` used for?

Reply:
Similar deal to NINO scope I believe

Reply:
Yep - it allows the CIC CRI to vary the text it displays to the user. Currently it says something like 'enter the details on your photo id', but this will make it display more like 'enter the details on your bank account'.